src/escalus_tcp.erl

```diff
@@ -54,6 +54,24 @@
 
 -define(WAIT_FOR_SOCKET_CLOSE_TIMEOUT, 200).
 -define(SERVER, ?MODULE).
+-define(DEFAULT_OPTS, [
```

Comment:
I'd define defaults as functions. That way it's way easier to trace in a running system what the defaults are, what the final options are, and also it's simple to do dirty tricks (read: manual testing) in the shell by adding `-compile(export_all).` and calling the function directly.

There's nothing in this code that steps outside valid, parseable Erlang, so using a macro is not required.